service/src/test/java/uk/gov/hmcts/reform/fpl/utils/ResourceLoader.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.document.domain.UploadResponse;
+
+import java.io.IOException;
+
+public class ResourceLoader {
+
+    private ResourceLoader() {
+    }
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    public static UploadResponse successfulDocumentManagementUploadResponse() throws IOException {
+        String response = new ResourceReader().read("responses/success.json");
```

Comment:
Let's reuse single instance of ResourceReader by declaring it as class field. Then here we would do:

```
resourceReader.read("responses/success.json");
```